### DISCUSSION TITLE
May I ask how to render this chord diagram separately

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#1142] [Q&A] [answered]
<img width="452" alt="image" src="https://user-images.githubusercontent.com/29226405/230857923-d66db9aa-6b0f-40c5-9253-a746bfef0264.png">
May I ask how to render this chord diagram separately

**Comment by hhly** [comment]
<img width="480" alt="image" src="https://user-images.githubusercontent.com/29226405/231062856-5fd91be9-a905-4d86-8666-c52f93ccd1b4.png">
And how can this lyrics change color together

**Comment by Danielku15** [comment, marked as answer]
> May I ask how to render this chord diagram separately

alphaTab currently does not provide a mechanism to render Chord diagrams separately to place them somewhere on the app/website. I added a new roadmap item to provide such a feature in future https://github.com/orgs/CoderLine/projects/12/views/1?pane=issue&itemId=25313077

> And how can this lyrics change color together
There is at this point also no setting to change the color of lyrics to something separate. They always have the `mainGlyphColor` or `secondaryGlyphColor`

https://alphatab.net/docs/reference/settings/display/resources/
